tests/snapd

```diff
@@ -14,13 +14,6 @@ lxd waitready --timeout=300
 # Check that preseed via snap works
 [ "$(lxc config get core.https_address)" = "[::]:8443" ]
 
-# LXD setup
-lxc storage create default dir
```

Comment:
yeah this kinds of defeats the purpose of the test to setup LXD in advance. Good spot.